test/Router/IntegrationTest.php

```diff
@@ -353,21 +360,25 @@ public function testMatchWithAllHttpMethods(string $adapter, string $method): vo
     }
 
     /**
-     * @psalm-return array<string, array{
-     *     0: class-string,
+     * @psalm-return Generator<array{
+     *     0: class-string<RouterInterface>,
      *     1: RequestMethod::METHOD_*
      * }>
      */
-    public function allowedMethod(): array
+    public function allowedMethod(): Generator
     {
-        return [
-            'aura-head'          => [AuraRouter::class, RequestMethod::METHOD_HEAD],
-            'aura-options'       => [AuraRouter::class, RequestMethod::METHOD_OPTIONS],
-            'fast-route-head'    => [FastRouteRouter::class, RequestMethod::METHOD_HEAD],
-            'fast-route-options' => [FastRouteRouter::class, RequestMethod::METHOD_OPTIONS],
-            'laminas-head'       => [LaminasRouter::class, RequestMethod::METHOD_HEAD],
-            'laminas-options'    => [LaminasRouter::class, RequestMethod::METHOD_OPTIONS],
-        ];
+        if (PHP_VERSION_ID < 80100) {
+            /**
+             * Excluded from PHP 8.1 onwards since the package seems to be abandoned.
+             */
+            yield [AuraRouter::class, RequestMethod::METHOD_HEAD];
```

Comment:
We're changing the suite functionality, with your changes. We previously had descriptions for the scenarios on this data provider, now that's gone. 

We should have something like this to keep the functionality (for all yields):

```suggestion
            yield 'aura-head' => [AuraRouter::class, RequestMethod::METHOD_HEAD];
```

BTW sorry for not sending all the comments on a single review. It's been a long long day

Reply:
no worries.